.travis/smoke-test.sh

```diff
@@ -3,13 +3,21 @@ set -ex
 
 COMPOSE_PROJECT_NAME="${COMPOSE_PROJECT_NAME:-sample-configuration}"
 HTTP_PORT="${HTTP_PORT:-8080}"
+HTTP_PORT_GATEWAY="${HTTP_PORT_GATEWAY:-8081}"
 
 echo "Wait for containers health"
+.scripts/docker/wait-healthy.sh "${COMPOSE_PROJECT_NAME}_scholarly-articles_fpm_1"
+.scripts/docker/wait-healthy.sh "${COMPOSE_PROJECT_NAME}_scholarly-articles_web_1"
+.scripts/docker/wait-healthy.sh "${COMPOSE_PROJECT_NAME}_api-gateway_1"
 .scripts/docker/wait-healthy.sh "${COMPOSE_PROJECT_NAME}_dummy-api_fpm_1"
 .scripts/docker/wait-healthy.sh "${COMPOSE_PROJECT_NAME}_browser_fpm_1"
 .scripts/docker/wait-healthy.sh "${COMPOSE_PROJECT_NAME}_pattern-library_1"
 .scripts/docker/wait-healthy.sh "${COMPOSE_PROJECT_NAME}_web_1"
 
+echo "Smoke testing api-gateway (scholarly-articles content-store)"
```

Comment:
Should maybe add a `/ping` to the gateway itself.

Reply:
It could also be used for the healthchecks (not sure if there is a spec for it)

Reply:
https://docs.google.com/document/d/1JhVpW8dxAuJqnk2YsQa76yxdIV_BlOd1QIi02fIu9mo/edit#heading=h.ik629cr6t76l